tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -86,6 +86,7 @@ def load_runner(self, url_protocol):
         self.parent.base.execute_script(self.runner_script % format_map)
 
     def close_old_windows(self):
+        self.webdriver.actions.release()
```

Comment:
While this seems sensible, I do wonder if we also need to do this elsewhere (e.g., after each test)?

Reply:
Ahh, I didn't know `testdriver.js` wasn't available in reftests.

I tagged James because you were away. You're of course more than welcome to review the change, Sam. Regarding this comment specifically -- do you know a better place to do this kind of cleanup? Logically, `Protocol.teardown` seems to make sense, but it's only called before killing WebDriver IIUC.

Reply:
I wasn't looking closely enough when I reviewed this before given I was trying to get through literally all the notifications. I'll try to look closer tomorrow.

Reply:
Hmmmm. The more I look at this the less happy I am with all our cleanup logic, bah! Uhhhh…

`Protocol.teardown` is at the very end when we close the session and restart the browser (or do as much as we can!). I guess we really want some sort of cleanup part to the protocol, ideally, and potentially make it configurable as to how much cleanup we do how often (closing windows, clearing actions, clearing cookies, etc.), really.

I think having it here is probably as good as anything else, really. Meh.